continuous_integration/recipe/meta.yaml

```diff
@@ -24,12 +24,15 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - setuptools-rust >=1.5.2
+    - libprotobuf                            # [build_platform != target_platform]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - setuptools-rust                        # [build_platform != target_platform]
```

Comment:
Think we may want to drop `setuptools-rust` from `build` after looking at [the `requirements` of another `setuptools-rust`-based package, `cryptography`]( https://github.com/conda-forge/cryptography-feedstock/blob/e70d84149adf02d7482113fea612ea2211a4b10c/recipe/meta.yaml#L61-L77 ) and only have it in `host`. Looks like [`maturin` does the same thing]( https://github.com/conda-forge/maturin-feedstock/blob/6528b97e83ca8d708f3300938cc1aa2eb0f86b02/recipe/meta.yaml#L18-L35 )

```suggestion
```

Reply:
Done below